docs/reference/cli.md

```diff
@@ -491,6 +471,33 @@ quickwit source add
 `--type` Type of the source. Available types are: `file` and `kafka`.    
 `--params` Parameters for the source formatted as a JSON object passed inline or via a file. Parameters are source-specific. Please, refer to the source's documentation for more details.    
 `--config` Quickwit config file.    
+
+*Examples*
+
+*Add a file source to `wikipedia` index*
+```bash
+ 
+quickwit source add --index wikipedia --id wikipedia-source --type file --params '{"filepath":"wiki-articles.json"}'
+
+```
+
+*Add a Kafka source to `wikipedia` index*
+```bash
+ 
+cat << EOF > wikipedia-source.json
+{
+  "topic": "wikipedia",
+  "client_params": {
+    "bootstrap.servers": "localhost:9092",
+    "group.id": "my-group-id",
+    "security.protocol": "SSL"
+  }
+}
+EOF
+quickwit source add --index wikipedia --id wikipedia-source --type kafka --params wikipedia-source.json
```

Comment:
I think replacing `id` with `source` would be more consistent
e.g.
`quickwit index create --index wikipedia` (not id)

Reply:
I had this discussion with @guilload but we decided to not touch it for now